fdbserver/include/fdbserver/LogSystem.h

```diff
@@ -853,8 +865,7 @@ void LogPushData::writeTypedMessage(T const& item, bool metadataMessage, bool al
 	for (auto& tag : next_message_tags) {
 		prev_tags.push_back(tag);
```

Comment:
What is the difference between "prev_tags" and "tags"? And, why is "tags" being used in place of "prev_tags" in the other/later commit?

Reply:
Also, are we sure that the "prev_tags"/"tags" that is being used here is exactly the same set of tags that the resolver uses too?

Reply:
in the later commit, `prev_tags` was sent as a parameter to the function into the local variable `tags`. `prev_tags` is a subset of the tags chosen in the resolver (see discussion above).

Reply:
> What is the difference between "prev_tags" and "tags"? And, why is "tags" being used in place of "prev_tags" in the other/later commit?

They are the same. I can remove `tags` and just use `prev_tags` as it is a class member.